src/gocept/selenium/tests/test_wd_selenese.py

```diff
@@ -245,6 +246,16 @@ def test_wait_for_retries_assertion_when_element_was_not_present(self):
             'Timed out after 1.0 s. ...Unable to locate element...',
             e.exception.msg)
 
+    def test_wait_for_retries_when_element_was_intercepted_in_click(self):
```

Comment:
I would prefer a rather structured test name like the next test in this file. The aim of the test has it's good place in the doc string.